plugins/doc_fragments/adguardhome.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+class ModuleDocFragment(object):
+
+    CONNECTIVITY = r'''
+options:
+  username:
+    description:
+      - AdGuard Home user.
+    required: true
+    type: str
+  password:
+    description:
+      - Related password for the AdGuard Home user
+    required: true
+    type: str
+  host:
+    description:
+      - IP or FQDN of AdGuard Home host.
+      - Requires also the protocol (http:// or https://).
```

Comment:
If it requires a protocol, it's neither an IP nor a FQDN, but an URL or URI. To make it clear, I would add an example:
```suggestion
      - URL of AdGuard Home host.
      - For example, V(https://my-adguard.my-domain) or V(http://192.168.1.2).
```